tests/L0/run_transformer/test_pipeline_parallel_fwd_bwd.py

```diff
@@ -48,12 +48,12 @@ def init_weights(m):
     return init_weights
 
 
-def get_target_loss_and_model(global_batch_shape: tuple, hidden_size: int, total_layers: int) -> float:  
+def get_target_loss_and_model(global_batch_shape: tuple, hidden_size: int, total_layers: int) -> Tuple[float, List[torch.nn.Module]]: 
```

Comment:
I guess `[w, b]` should be `List[torch.Tensor]`?
```suggestion
def get_target_loss_and_model(global_batch_shape: tuple, hidden_size: int, total_layers: int) -> Tuple[float, List[torch.Tensor]]: 
```

Reply:
I shall add that `loss` is a `torch.Tensor` of single element.